src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -4,30 +4,28 @@
 
 namespace Microsoft.CodeAnalysis.Serialization;
 
-internal enum WellKnownSynchronizationKind
+internal enum WellKnownSynchronizationKind : byte
 {
     // Start at a different value from 0 so that if we ever get 0 we know it's a bug.
 
     // Solution snapshot state, including generator info (like frozen generated documents).
     SolutionCompilationState = 1,
 
     // Solution snapshot state, only referencing actual user (non-generated) documents, options, and references.
-    SolutionState,
-    ProjectState,
-
-    ChecksumCollection,
```

Comment:
this value was not actualy used in practice. we never have OOP ask for the 'ChecksumCollectoin' at any point.  rather, those are just data-children in larger objects (like ProjectStateChecksums) that are returned back.